scripts/models/TownHall.js

```diff
@@ -157,6 +157,9 @@
         // Currently some of the data is inconsistent.  Some parties are listed as "Democrat" and some are listed as "Democratic", etc
         // TODO:  Once data is sanatized use return TownHall.filters[key].indexOf(townhall[key]) !== -1;
         return TownHall.filters[key].some(function(filter) {
+          if (filter === 'Democrat' || filter === "Democratic" || filter === "Republican" || filter === "Independent") {
```

Comment:
why not just do if (key === party?)

Reply:
it also seems like you should make the values of the dropdowns to be 'D' and 'R' etc right?

Reply:
https://github.com/townhallproject/townHallProject/blob/master/index.html#L569

Reply:
It's not necessary to do the 'D' or 'R' or 'I' change in the html, but we can do it that way it we want to. My thinking was that we keep the html the same, so when someone selects 'Republican' as their filter option, the tag that displays also says 'Republican' rather than just 'R' but either way can work.

Reply:
(Apologies if I'm misinterpreting here)

Would prefer that the drop down continue to publicly display "Republican" rather than "R"

Reply:
i meant the data value, not what the drop down display.

Reply:
Ok, it should be updated now.

I was thinking of the id as well @meganrm - I thought the table buttons were being generated by the id instead of the filter text, so that the dropdown read 'Democratic' but the filter btn read 'D' but it looks like both are fine.

Reply:
The id is what gets added as a filter, so you could change them to the single letter and then remove the if (key === party) check too.